core/src/main/java/org/apache/accumulo/core/client/admin/DiskUsage.java

```diff
@@ -47,9 +48,9 @@ public boolean equals(Object o) {
 
     DiskUsage diskUsage = (DiskUsage) o;
 
-    if (tables != null ? !tables.equals(diskUsage.tables) : diskUsage.tables != null)
+    if (!Objects.equals(tables, diskUsage.tables))
       return false;
-    return usage != null ? usage.equals(diskUsage.usage) : diskUsage.usage == null;
+    return Objects.equals(usage, diskUsage.usage);
```

Comment:
This could be reduced further using AND, something like:
`return Objects.equals(tables, diskUsage.tables) && Objects.equals(usage, diskUsage.usage);`

Also, any thought to replacing hashCode at the same time to use Objects.hash(tables, usage); - that might help with the constraint that equals and hashCode are consistent.

Reply:
Thanks @EdColeman . Updated to add further simplification and updated hashCode.

Reply:
I think the same thing could apply to the other 2 classes.

Reply:
@EdColeman pushed up changes to the other two classes.